rust/agama-network/src/nm/client.rs

```diff
@@ -266,18 +291,33 @@ impl<'a> NetworkManagerClient<'a> {
         let path = if let Ok(proxy) = self.get_connection_proxy(conn.uuid).await {
             let original = proxy.get_settings().await?;
             let merged = merge_dbus_connections(&original, &new_conn)?;
-            proxy.update(merged).await?;
+            // https://networkmanager.dev/docs/api/latest/nm-dbus-types.html#NMSettingsConnectionFlags
+            // 0x1 persist to disk, 0x8 memory only
+            let persist = if conn.keep { 0x1 } else { 0x8 };
```

Comment:
master: it would be nice to keep to use constants with good names for those flags.